### PR TITLE
Add well supported devices to readme

### PR DESCRIPTION
**Pull request by bbrown1867**
This PR implements the documentation update discussed in #462

### CODE REVIEW

**Comment by therealprof** [comment]
Sounds good to me, and I just noticed we should rewrite that whole section anyway since `embedded-hal` is an official part of the WG and not a community effort...

@rust-embedded/all Opinions?

File: README.md [hunk 1]
```diff
@@ -482,6 +482,17 @@ These are other projects you may be interested in but that (currently) are not o
 
 ## On going community efforts
 
+### Device specific communities
+
+Several device specific communities exist that are not part of the working group.
+These communities maintain crates for peripheral access, hardware abstraction,
+examples, and more that are specific to a particular family of devices.
+Some of the higher profile ones are listed below.
```
Comment (therealprof) [inline README.md:490]:
```suggestion
```

I don't think we need to claim these are higher profile, there're also plenty other ones around. Maybe encourage people to add them here instead?

**Comment by jonas-schievink** [comment]
We could also make the "awesome embedded Rust" project more visible in the readme and add a section with "bigger" vendor support projects there (eg. anything that is its own GitHub org)

**Comment by Disasm** [comment]
Irrelevant to this PR, I think it's a good idea to have a "hal x peripherals" table with a support level for different peripherals somewhere so that people can match their expectations with the reality before they dig too deep. It will be quite difficult to maintain, though.

File: README.md (hunk 1 above)
Comment (bbrown1867) [reply README.md]: Agreed, fixed.

**Comment by bbrown1867** [comment]
@therealprof I am happy to remove the `embedded-hal` blurb in this section (2 birds, 1 stone). I noticed it was stale also. What about  `The weekly driver initiative` is that still active?

**Comment by therealprof** [comment]
> We could also make the "awesome embedded Rust" project more visible in the readme and add a section with "bigger" vendor support projects there (eg. anything that is its own GitHub org)

I agree in general, though we're already kind of violating the spirit of "awesome lists" with our interpretation of the rules: https://github.com/sindresorhus/awesome/blob/main/awesome.md

I don't mind mentioning the community driven orgs here as an entry point for interested people so they end up quicker at the right place without shying them away with the information overload on aer. ;)

**Comment by eldruin** [comment]
Yes. I agree the embedded-hal section should definitely not be listed as a community effort.
Some people post every now and again to the weekly driver initiative. Although I agree it is kind of stale, there are a lot of people subscribed so driver announcements can reach a lot of interested people at once. The awesome embedded rust repo is much more active, though.

**Comment by therealprof** [comment]
> I am happy to remove the `embedded-hal` blurb in this section (2 birds, 1 stone). I noticed it was stale also. 

Please let's keep those topics separate. You're very welcome to propose another PR with that change, though. That one is non-controversial. ;)

> What about `The weekly driver initiative` is that still active?

Could use some love and drive but in theory yes.

File: README.md [hunk 2]
```diff
@@ -482,6 +482,17 @@ These are other projects you may be interested in but that (currently) are not o
 
 ## On going community efforts
 
+### Device specific communities
+
+Several device specific communities exist that are not part of the working group.
+These communities maintain crates for peripheral access, hardware abstraction,
+examples, and more that are specific to a particular family of devices.
+The list below is not exhaustive and should be updated as device support increases.
```
Comment (adamgreig) [inline README.md:490]:
```suggestion
The list below is not exhaustive and will be updated as device support increases.
```

File: README.md [hunk 3]
```diff
@@ -482,6 +482,17 @@ These are other projects you may be interested in but that (currently) are not o
...
+
+- [nrf-rs](https://github.com/nrf-rs): Nordic nRF series of microcontrollers
+- [stm32-rs](https://github.com/stm32-rs): STM 32-bit microcontrollers
```
Comment (adamgreig) [inline README.md:493]:
```suggestion
- [stm32-rs](https://github.com/stm32-rs): STM32 microcontrollers
```

Comment (bbrown1867) [reply README.md]: Fixed, thanks.

File: README.md (hunk 2 above)
Comment (bbrown1867) [reply README.md]: Fixed, thanks.

**Pull request merged**